src/args.rs

```diff
@@ -225,6 +225,18 @@ pub struct CliArgs {
     )]
     pub web_upload_concurrency: usize,
 
+    /// chmod permissions of uploaded files
+    ///
+    /// This takes an octal number that is similar to the one the chmod use
+    #[cfg(unix)]
+    #[arg(
+        long = "chmod",
+        value_parser(parse_file_mode),
+        env = "MINISERVE_CHMOD",
+        default_value = "0600"
+    )]
+    pub chmod: u32,
```

Comment:
This doesn't make make sense without `-u` right? So let's add `requires = "allowed_upload_dir",`